### ISSUE TITLE
Release for Visual Studio 2022

### ISSUE THREAD

**Issue by xperiandri** [MisterJimson/editProjFiles#8]

**Comment by MisterJimson** [comment]
Wow I can't believe this is still in use.

Newer versions of VS allow editing the project file with a right click, is there a reason that doesn't work for you in VS 2022?

**Comment by xperiandri** [comment]
Yes, I constantly use it when develop Uno apps. It is a crucial tool for UWP, Xamarin.Android, XamariniOS, and Xamarin.Mac development

**Comment by xperiandri** [comment]
This has to be fixed with .NET 6 and MAUI release but until then it is very hard to work without it

**Comment by kkm000** [comment]
@MisterJimson,

> Newer versions of VS allow editing the project file with a right click, is there a reason that doesn't work for you in VS 2022?

Not exactly and yes, the reason is I'm using C++. :-(
